lib/simplifyExpression/index.js

```diff
@@ -1,7 +1,9 @@
 const math = require('mathjs');
 const stepThrough = require('./stepThrough');
+const Substitute = require('../Substitute');
 
-function simplifyExpressionString(expressionString, debug=false) {
+function simplifyExpressionString(expressionString, debug=false, scope={}) {
```

Comment:
it'd be nice if debug was the last argument, though I know that's a lot of moving code around so let me know if you want help going through and editing it all